package.xml

```diff
@@ -36,6 +36,8 @@
  </notes>
  <contents>
   <dir baseinstalldir="/" name="/">
+   <file name="LICENSE" role="doc" />
+   <file name="README.rst" role="doc" />
```

Comment:
`README.rst` is just a symlink to `docs/guide.txt`, so I don't think we need to include it in the package twice.

We could make `README.rst` the canonical file and get rid of the `docs/guide.txt` file entirely, though.